packages/components/src/composite/stories/utils.tsx

```diff
@@ -6,8 +6,11 @@ import type { StoryContext } from '@storybook/react';
 /**
  * Internal dependencies
  */
-import type { CompositeStoreProps } from '..';
+import type { StoreProps as CompositeStoreProps } from '..';
 
+/**
+ * Renders a composite widget.
+ */
```

Comment:
I think we also need a code snippet for the Storybook docs.

```suggestion
/**
 * Renders a composite widget.
 *
 * ```jsx
 * const useCompositeStore = useCompositeStore();
 * const store = useCompositeStore();
 * <Composite store={store}>
 *   <Composite.Item>Item 1</Composite.Item>
 *   <Composite.Item>Item 2</Composite.Item>
 * </Composite>
 * ```
 */
```